src/server/plugins/blankie.test.ts

```diff
@@ -1,18 +1,52 @@
-import pluginBlankie from '~/src/server/plugins/blankie.js'
+import { config } from '~/src/config/index.js'
+import { configureBlankiePlugin } from '~/src/server/plugins/blankie.js'
 
 describe('Server Blankie Plugin', () => {
-  test('configuration options are correct', () => {
-    const { options } = pluginBlankie
+  test('configuration default options are provided', () => {
+    config.set('googleAnalyticsTrackingId', '')
+
+    const { options } = configureBlankiePlugin()
 
     expect(options).toEqual({
       defaultSrc: ['self'],
       fontSrc: ['self', 'data:'],
       frameSrc: ['self', 'data:'],
       connectSrc: ['self'],
-      scriptSrc: ['self', 'unsafe-inline'],
+      scriptSrc: ['self', 'strict-dynamic', 'unsafe-inline'],
```

Comment:
This line throws CSP errors in older Safari:
https://caniuse.com/?search=strict-dynamic

All versions up to (but not including) 15.4 from March 2022 😬

Sorry, it's too much of an impact and will break the **Accessible Autocomplete** component

Could we add a new `{{ getAssetHash() }}` helper to read from the webpack JSON manifest?

Reply:
Older Safari just throws CSP errors, we might be OK simply because it can't understand it

But we should verify this really